README.md

```diff
@@ -2,16 +2,16 @@
 
 [![Build Status](https://travis-ci.org/igneus/church-calendar-api.svg?branch=master)](https://travis-ci.org/igneus/church-calendar-api)
 
-Browse Roman Catholic liturgical calendar through a web interface
-or obtain it's data in JSON format through an API.
+Browse the Roman Catholic liturgical calendar through a web interface
```

Comment:
I suggested the definite article here because it is specific kind of calendar (there is only one liturgical calendar, that which the Vatican has approved). 

For ref: https://owl.purdue.edu/owl/general_writing/grammar/using_articles.html#:~:text=The%20definite%20article%20is%20used,the%20dog%20that%20bit%20me.

Reply:
> there is only one liturgical calendar, that which the Vatican has approved

There are many Roman Catholic liturgical calendars (General Roman Calendar, calendars of nations, provinces, dioceses, religious orders and their provinces and houses, calendars of single churches, ...) and the API can provide access to many of them, so I'm not sure that the definite article is appropriate here.

Reply:
Ah okay, in that case, it should be 'Roman Catholic Liturgical calendars', plural and without the definite article.